2017/presentations/with_python_security/text.md

```diff
@@ -0,0 +1,519 @@
+# with python: security() - Piotr Dyba
+
+
+#### Disclaimers:
+The views and opinions expressed in this article focus on the developer's perspective an are solely those of the author.
+I have received no payments for mentioning any of the open-source or commercial products mentioned in this article.
+
+
+## What is cybersecurity and why it's so important nowadays?
+
+Cybersecurity encompasses means and protocols to defend resources and devices (computer systems, IoT devices, smartphones) from disruption, damage or theft.
+It is not only about firewalls and antivirus software, but rather it starts with the hardware,
+including policies for handling computers and servers at a company,
+access rights to server rooms and even which devices are allowed to be brought into the company building. For instance, it is common to ban external pen-drives.
+To be able to assume that our hardware is safe, we need to harden our software on both workstations and servers and impose
+proper policies for the users, data handling, access.
+
+[Data leaks in time](http://www.informationisbeautiful.net/visualizations/worlds-biggest-data-breaches-hacks/)
+
+Looking at the number and severity of leaks over time, it is easy to observe a huge increase in their occurences over the years and severity rising from 1 000 to above 1 000 000 000 user's data.
+
+![00 NotPetya maleware screen](./000_notpetya.png)
+
+Remember that screen from just two weeks ago? NotPetya laid waste on many companies across the globe in just one day, some of which are still recovering.
+The most known cases were companies such as TNT and Raben.
+Raben is a good example of well prepared recovery that took less than 24 hours. In that time, Raben moved their whole server infrastructure from Windows to Linux, which was quite an achievement.
+
+![01 NotPetya maleware screen](./001_attacks_per_day.png)
+
+The graph above shows the number of attackers per day in the last 6 months, which varied from 2 to almost 16 millions per day.
+An attack means an action against one of F-secure's research honeypots deployed on the public internet.
+A honeypot is a server that pretends to be an easy target for an attacker, where all his or her actions can be monitored
+and registered, but which crucially cannot affect the critical parts of the broader infrastructure.
+It’s just a trap that exposes services like ssh, mysql, smtp etc., in our case written in pure python.
```

Comment:
Może zamiast "our case" dać "this case", bo our może brzmieć delikatnie zbyt stronniczo